admin-notice/class-admin-notice-controller.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Automattic\VIP\Admin_Notice;
+
+// function sample_admin_notice__error() {
+//     $class = 'notice notice-info';
+//     $message = __( 'Irks! An error has occurred.', 'sample-text-domain' );
+
+//     printf( '<div class="%1$s"><p>%2$s</p></div>', esc_attr( $class ), esc_html( $message ) );
+// }
+// add_action( 'admin_notices', 'sample_admin_notice__error' );
+
+
+require_once __DIR__ . '/class-admin-notice.php';
```

Comment:
Keep class definition and side-effect code to their own files. Move this require into the admin-notices/admin-notices.php file. The responsibility of this file is to define the class, and not to ensure other files are loaded.

Reply:
Done